### PR TITLE
finished last file LoginGUI.java

### PR DESCRIPTION
**Pull request by S4kRiPp4**
-Spellchecked all files
-Spaced variable declarations in most files to remain consistent
-deleted/added whitespace
-few classes still have multiple declaration in one line but its whatever

-finished formating LoginGUI.java
-went through all files one last time to make small corrections

### CODE REVIEW

**Comment by S4kRiPp4** [comment]
Was moved to draft and was waiting for Jordan's PR to merge since we were both making changes to the same file. Merge conflicts are resolved, I pulled local too w/ checkout to make sure everything still works. Ready for review.

File: app/src/main/java/com/yagni/view/LoginGUI.java [hunk 1]
```diff
@@ -112,14 +198,44 @@ public class LoginGUI implements ActionListener {
...
+		} else {
+			success.setText("Invalid credentials! Please Try Again. ");
+			success.setForeground(Color.RED); 	
+		}
+=======
         // display all content to the GUI
         frame.setVisible(true);
 
     }
+>>>>>>> main
```
Comment (chillyssa) [inline app/src/main/java/com/yagni/view/LoginGUI.java]: Looks like you have some incoming changes from Head and your working branch that are conflicting. Various points in this whole file are affected.

**Review by chillyssa — CHANGES_REQUESTED** [review]
LoginGUI needs to be reviewed again. There are several points where there are incoming changes from head that conflict with you main working branch. Did these not cause errors for you locally @S4kRiPp4 ?

File: app/src/main/java/com/yagni/view/LoginGUI.java (hunk 1 above)
Comment (S4kRiPp4) [reply app/src/main/java/com/yagni/view/LoginGUI.java]: It said I resolved all them locally but I'll go through it again w/ you to be certain.

**Review by chillyssa — APPROVED** [review]
Thank you for making those updates and looking at the conflicts! 

Looks good to me!

**Review by snowjtSD — APPROVED** [review]
nice clean up looks good and worked on my end so nothing broken i could see

**Pull request merged**